prowler/lib/utils/utils.py

```diff
@@ -80,31 +81,57 @@ def hash_sha512(string: str) -> str:
     return sha512(string.encode(encoding_format_utf_8)).hexdigest()[0:9]
 
 
-def detect_secrets_scan(data):
-    temp_data_file = tempfile.NamedTemporaryFile(delete=False)
-    temp_data_file.write(bytes(data, encoding="raw_unicode_escape"))
-    temp_data_file.close()
+def is_known_non_secret(line):
+    known_non_secrets = [
+        "MYSQL_ALLOW_EMPTY_PASSWORD",
+        "GCP_SECRET_ARN",
+        "S3_BUCKET",
+        "AWS_STACK",
+    ]
```

Comment:
I think this list could end up with thousands of values, I'm not sure about placing it here. Do we have any other way not to flag them as secrets?

Reply:
I think I'll remove that for now until we explore all the options to try not to flag them as secrets.

Reply:
Let's try to improve the configuration for the test that will fail once we remove this block.